### ISSUE TITLE
Buttons in navigation bar doesn’t work on small screen

### ISSUE THREAD

**Issue by jakubknejzlik** [crudlio/crudl#44]
When using small screen width (tested on iPhone 7 portrait mode) I can see the buttons in navigation bar (search/add icons) but tapping on them doesn’t do anything. 
After rotating to landscape, buttons work as expected.

**Comment by parsch** [comment]
Yes, that's quite possible. Though the current version looks quite responsive there are actually no distinctive responsive styles (this needs to be improved).
